### PR TITLE
[FLINK-13086]add Chinese documentation for catalogs

### PR DESCRIPTION
**Pull request by yiduwangkai**
## What is the purpose of the change

*Support Chinese version of catalog.md *

## Brief change log

*(for example:)*
  - *The TaskInfo is stored in the blob store on job creation time as a persistent artifact*
  - *Deployments RPC transmits only the blob storage reference*
  - *TaskManagers retrieve the TaskInfo from the blob cache*

## Verifying this change

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (100MB)*
  - *Extended integration test for recovery after master (JobManager) failure*
  - *Added test that validates that TaskInfo is transferred only once across recoveries*
  - *Manually verified the change by running a 4 node cluser with 2 JobManagers and 4 TaskManagers, a stateful streaming program, and killing one JobManager and two TaskManagers during the execution, verifying that recovery happens correctly.*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / no)
  - The serializers: (yes / no / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / no / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (yes / no / don't know)
  - The S3 file system connector: (yes / no / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: docs/dev/table/catalog.zh.md [hunk 1]
```diff
@@ -186,53 +187,57 @@ Currently `HiveCatalog` supports most Flink data types with the following mappin
...
 
-To use `HiveCatalog` in Flink Table API and SQL, users need to include `flink-connector-hive` jar in their projects.
+要在Flink Table API和SQL中使用`HiveCatalog`，用户需要在他们的项目中包含`flink-connector-hive` jar。
 
 {% highlight xml %}
 <dependency>
-	<groupId>org.apache.flink</groupId>
-	<artifactId>flink-connector-hive_{{ site.scala_version_suffix }}</artifactId>
-	<version>{{site.version}}</version>
+    <groupId>org.apache.flink</groupId>
```
Comment (bowenli86) [inline docs/dev/table/catalog.zh.md]: revert changes to this code snippet as it doesn't do anything except indentation

File: docs/dev/table/catalog.zh.md [hunk 2]
```diff
@@ -307,9 +312,9 @@ myHive1.listPartitions(myTablePath);
 
 
 
-## Use HiveCatalog in Flink SQL Client
+## 在Flink SQL Client中使用HiveCatalog
 
-Users can specify catalogs in the yaml config file of Flink SQL CLI. See [SQL Client]({{ site.baseurl }}/dev/table/sqlClient.html) for more details.
+用户可以在Flink SQL CLI的yaml配置文件中指定目录。 具体请参阅 [SQL Client]({{ site.baseurl }}/dev/table/sqlClient.html) for more details.
```
Comment (bowenli86) [inline docs/dev/table/catalog.zh.md]:
```suggestion
用户可以在Flink SQL CLI的yaml配置文件中指定目录。 具体请参阅 [SQL Client]({{ site.baseurl }}/dev/table/sqlClient_zh.html) for more details.
```

File: docs/dev/table/catalog.zh.md [hunk 3]
```diff
@@ -357,7 +362,7 @@ Flink SQL> ......
 
 ```
 
-For a full list of Flink SQL commands to access Hive meta-objects, see [FLINK SQL]({{ site.baseurl }}/dev/table/sql.html)
+有关访问Hive元对象的Flink SQL命令的完整列表，请参阅 [FLINK SQL]({{ site.baseurl }}/dev/table/sql.html)
```
Comment (bowenli86) [inline docs/dev/table/catalog.zh.md]:
```suggestion
有关访问Hive元对象的Flink SQL命令的完整列表，请参阅 [FLINK SQL]({{ site.baseurl }}/dev/table/sql_zh.html)
```

**Comment by wuchong** [comment]
hi @yiduwangkai , why we removed `sqlClient.zh.md` page? Is that a mistake?

**Comment by yiduwangkai** [comment]
@wuchong sqlClient.zh.md split to another issue  https://github.com/apache/flink/pull/9164

**Comment by yiduwangkai** [comment]
i have updated @bowenli86

**Comment by wuchong** [comment]
@yiduwangkai , please rebase your commit on the latest master and do not use "merge".

**Comment by yiduwangkai** [comment]
yes i used rebase at first, But when I execute this command, when I push,
git asks for pull, maybe it is improper operation before, causing the
situation to be merged when the pull occurs, but please be assured that no
other files have been changed. thx

Jark Wu <notifications@github.com> 于2019年7月21日周日 下午10:19写道：

**Comment by yiduwangkai** [comment]
Why is there a new merge commit extra information, because I use the pull
command to pull the latest code of the upstream branch, this record will be
generated if the local branch lags behind the upstream branch

kai wang <yiduwangkai@gmail.com> 于2019年7月21日周日 下午11:28写道：

**Comment by bowenli86** [comment]
Hi @yiduwangkai , can you rebase this PR?

**Comment by yiduwangkai** [comment]
> can you rebase this PR?

oK

**Comment by yiduwangkai** [comment]
OK

Bowen L <notifications@github.com> 于2019年8月13日周二 上午2:36写道：

**Comment by yiduwangkai** [comment]
i want to close this pr, merge [apache/flink] [FLINK-13086]add Chinese
documentation for catalogs (#9163)
and [apache/flink] [FLINK-12894]add Chinese documentation of how to
configure and use catalogs in SQL CLI (#9164) together and submit a new
one, do you agree
kai wang <yiduwangkai@gmail.com> 于2019年8月13日周二 上午9:32写道：

**Comment by bowenli86** [comment]
opening a single PR with multi commits sounds good to me

**Pull request closed without merging**